frontend/src/pages/manage-member/ManageMember.jsx

```diff
@@ -74,64 +75,74 @@ const ManageMember = () => {
     },
   ];
 
-  const handleSearchMemberByName = (value) => {
-    setSearch(value.target.value);
-    setMemberTypeFilter(null);
-    setIscoFilter(null);
-    const filteredByMemberName = data.filter((m, i) => {
-      return m.name.toLowerCase().includes(value.target.value.toLowerCase());
-    });
-    setMemberNameFilter(filteredByMemberName);
+  useEffect(() => {
+    setPage(1);
+  }, []);
+
+  const handleMemberNameFilter = (val) => {
```

Comment:
Case: 
- going to page 2, then filter with organisation/member/isco dropdown, this will send a request with page 2 into backend. I think better if we reset the page to 1 when filtering.